app/uk/gov/hmrc/tradergoodsprofilesrouter/models/request/eis/advicerequests/RequestEisAdviceRequest.scala

```diff
@@ -14,24 +14,24 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.tradergoodsprofilesrouter.models.request.eis.accreditationrequests
+package uk.gov.hmrc.tradergoodsprofilesrouter.models.request.eis.advicerequests
 
 import play.api.libs.json.{Json, OFormat}
 
-case class RequestEisAccreditationRequest(
-  accreditationRequest: AccreditationRequest
+case class RequestEisAdviceRequest(
+  adviceRequest: AdviceRequest
 )
```

Comment:
Has this been confirmed to change from EIS? It may the app to break on test environment

Reply:
Yeah good point, I don't think it has been confirmed. I'll check